lib/streamlit/runtime/caching/memo_decorator.py

```diff
@@ -269,9 +270,10 @@ def __call__(
             for an unbounded cache. (When a new entry is added to a full cache,
             the oldest cached entry will be removed.) The default is None.
 
-        ttl : float or None
+        ttl : float or timedelta or None
             The maximum number of seconds to keep an entry in the cache, or
-            None if cache entries should not expire. The default is None.
+            None if cache entries should not expire. If timedelta object, total number
+            of seconds will be used. The default is None.
```

Comment:
> If timedelta object, total number of seconds will be used.

Maybe better wording could be used here

Reply:
Is there ambiguity here? I think we can maybe just leave out the "If timedelta object, total number of seconds will be used" bit of the comment (i.e. just revert to its previous text), because it seems obvious from the context.